internal/nginx/manager.go

```diff
@@ -35,7 +35,7 @@ const (
 	appProtectAgentStartCmd  = "/opt/app_protect/bin/bd_agent"
 
 	// appPluginParams is the configuration of App-Protect plugin
-	appPluginParams = "tmm_count 4 proc_cpuinfo_cpu_mhz 2000000 total_xml_memory 307200000 total_umu_max_size 3129344 sys_max_account_id 1024 no_static_config"
+	appPluginParams = "tmm_count 4 proc_cpuinfo_cpu_mhz 2000000 total_xml_memory 471859200 total_umu_max_size 3129344 sys_max_account_id 1024 no_static_config"
```

Comment:
What's the purpose of this change?

Reply:
This is to give enough memory available for xml parser in order to avoid issuers with big policies. https://docs.nginx.com/nginx-app-protect/admin-guide/install/#:~:text=file.%20The%20number-,471859200,-should%20be%20enough

Reply:
it's not really relevant to the reconnect period, is it?

Reply:
agree. create separate PR?

Reply:
yeah I think it makese sense

Reply:
created PR https://github.com/nginxinc/kubernetes-ingress/pull/2455